rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -7,6 +7,21 @@ function _M.get_backends_data()
   return configuration_data:get("backends")
 end
 
+function _M.new_backends()
```

Comment:
I think the convention is to name these kind of functions `_handler`, no? i.e `backends_handler` in this case.

--

Also since you are using this only locally, you don't have to export it. You can do
```
local function backends_handler()
...
```

Reply:
Oh wait this is just for extracting request body. IMO we should not make it backends specific - there's nothing in the function body that's backends specific - this should just be a helper function that returns request body and `nil` or in case of error `nil` and error message